src/main/java/org/synyx/urlaubsverwaltung/application/service/ApplicationRecipientService.java

```diff
@@ -84,6 +86,31 @@
         return concat(bosses, relevantBosses, responsibleDepartmentHeads);
     }
 
+    /**
+     * Returns the relevant recipients for a given person.
+     * <p>
+     * If the person is in a department than return
+     * - the persons with the role department head for this user
+     * else return
+     * - all persons with the notification type {@code NOTIFICATION_OFFICE}
+     *
+     * @param application that has been rejected
+     * @return all relevant recipients to inform
+     */
+    List<Person> getRelevantRecipients(Application application) {
```

Comment:
Does this method pay attention for two stage approval?

Reply:
I think it should or? Who should get the information on the two stage approval?

Reply:
hm or maybe not :)

Reply:
We should take a deeper look at that. The second stage authority would not receive any information. We should change that.